### ISSUE TITLE
Python 2.7 code generator

### ISSUE THREAD

**Issue by marcb** [asyncapi/asyncapi#23]
In many enterprise environments there are restrictions on what agents can be written in… sometimes Node.js isn't allowed.

**Comment by fmvilas** [comment]
Agree. Do you think it's something you can help with? We don't know much about Python :)

**Comment by marcb** [comment]
Most probably. Customer's project I am on will be integrating AMQP soon, and the broader organisation is showing more and more interest in async APIs (how I discovered https://www.asyncapi.com via internal thread).

Another option is to leverage the Go initiative and support embedding aka https://docs.python.org/2/extending/extending.html.

Although my personal interests are Rust oriented, but I haven't absorbed the spec enough to see if a strongly typed approach is an advantage or a challenge.

All that said, many things on my plate…

**Comment by fmvilas** [comment]
Great. I can help you regarding the spec if you decide to contribute. Also if you want to chat join us on Slack https://async-apis-slack.herokuapp.com/

**Comment by fmvilas** [comment]
@marcb do you think this is something we can revive? I'll be glad to help as much as possible.

**Comment by AtosNicoS** [comment]
Python2 will be dead in 2020, I do not recomment "wasting time" into implementing a python2 generator. However a Python3 version would be extremely nice, as it is one of the most used IoT Languages.

**Comment by fmvilas** [comment]
Yeah, agree. I'd be very happy if someone can help with it. It doesn't need to be perfect. Think that most people use code generators just for prototyping ideas.
